.github/workflows/deploy.yml

```diff
@@ -12,9 +12,6 @@ jobs:
     with:
       app_name: ${{ github.event.repository.name }}
       api: true
-      users: 2
-      spawn-rate: 2
-      run-time: 11s
```

Comment:
Can you re-add users, run-time and spawn-rate please? @gio-karunakaran

Some repos needs to have different locust configs for the performance tests. This will override what we have set in the workflows repo for this repo only.

Reply:
Spoke with Rayhan. Agreed to create a new ticket to add mentioned vars back on if needed.